airflow/www/templates/appbuilder/navbar.html

```diff
@@ -21,6 +21,13 @@
 {% set languages = appbuilder.languages %}
 
 <div class="navbar navbar-fixed-top" role="navigation" style="background-color: {{ navbar_color }};">
+  {% if navbar_text_color %}
+    <style>
+      .navbar .navbar-collapse .navbar-nav a {
```

Comment:
```suggestion
      .navbar-nav > li > a {
```

This is probably a better selector to use.